app/assets/javascripts/google-analytics.js

```diff
@@ -0,0 +1,47 @@
+// Tracks special user actions with Google Analytics
```

Comment:
Good call.  I'll change the file name to something more appropriate.  I'll also include a comment in this file about the overall Google Analytics tracking setup in the app.